src/renderers/shaders/ShaderChunk/packing.glsl

```diff
@@ -6,17 +6,26 @@ vec3 unpackRGBToNormal( const in vec3 rgb ) {
   return 1.0 - 2.0 * rgb.xyz;
 }
 
+const float PackUpscale = 256. / 255.; // fraction -> 0..1 (including 1)
+const float UnpackDownscale = 255. / 256.; // 0..1 -> fraction (excluding 1)
+
+const vec3 PackFactors = vec3( 256. * 256. * 256., 256. * 256.,  256. );
+const vec4 UnpackFactors = UnpackDownscale / vec4( PackFactors, 1. );
+
+const float ShiftRight8 = 1. / 256.;
+
+vec4 packDepthToRGBA( const in float v ) {
+
+	vec4 r = vec4( fract( v * PackFactors ), v );
```

Comment:
See here for my attempt to make this optimization (fract versus mod):  https://github.com/mrdoob/three.js/pull/8547/files/272056d01bc6325de7d8c306a174b47e634fb918..8dfb3f1d2a53d389540bb514d5fc183605e9b133#r58754365

This refactoring of packing shouldn't really change speed, const floats inside of the function or outside of the function are still pre-computed by the compiler I think.